force-app/main/default/classes/TestCreateOffer.cls

```diff
@@ -0,0 +1,58 @@
+/**
+ * @author Arkadiusz Bednarz <arkadiusz.bednarz@accenture.com>
+ * @date 29/12/2021
+ * @description test class for Create_offer trigger.
+ */
+@isTest
+public class TestCreateOffer {
+    
+     /**
+      * @author Arkadiusz Bednarz <arkadiusz.bednarz@accenture.com>
+      * @date 29/12/2021
+      * @description method checks the creation of new offers and blocking the creation of 2 physical offers for 1 Vehicle__c
+      */
+      @isTest 
+      static void TestAddPhysicalOfferWhereOfferExist(){
```

Comment:
testy dla trigerow powiny byc bulkowe tzn na 200 elementach wtedy jak jest jakis select w petli albo cos innego to wyskoczy blad